newa/__init__.py

```diff
@@ -31,9 +31,14 @@
 import ruamel.yaml.representer
 import urllib3.response
 from attrs import define, field, frozen, validators
+from jira import JIRA, JIRAError
+from jira import Issue as JiraIssue
```

Comment:
Just a thought but I find the amount of Jira/Issue related classes already confusing and having hard time understanding where it comes from. Would it make sense to import only `jira` module so that references like `jira.Issue` would be more "informative"?

Reply:
Yup, that sounds like a good idea.